src/App.js

```diff
@@ -1,23 +1,132 @@
-import logo from './logo.svg';
-import './App.css';
+import "./styles/App.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+import CitySelectionForm from "./components/CitySelectionForm";
+import DisplayInformation from "./components/DisplayInformation";
+import DisplayPhotos from "./components/DisplayPhoto";
 
 function App() {
+  //Hardcoded to test Display
+  // const cityInfo = {
+  //   name: "Toronto",
+  //   country: "Canada",
+  //   population: 200000,
+  //   latitude: 51.05,
+  //   longitude: -114.067,
+  //   is_capital: false,
+  // };
+
+  //States
+  // const [city, setCity] = useState("Rio de Janeiro");
+  const [city, setCity] = useState("");
+  const [selectedInfo, setSelectedInfo] = useState({
+    // name: "Toront",
+    // country: "Canada1",
+    // population: 200000,
+    // latitude: 51.05,
+    // longitude: -114.067,
+    // is_capital: false,
+  });
+  const [cityPhotos, setCityPhotos] = useState([]);
+
+  //API call
+  useEffect(() => {
+    //1st Call: Ninjas City API
+
+    if (city) {
+      axios({
+        url: `https://api.api-ninjas.com/v1/city?name=${city}`,
+        method: "GET",
+        headers: {
+          "X-Api-Key": "OMPD0sHj09N5XJV/prIM3Q==zKLddhyIL406cdHa",
+        },
+      }).then((res) => {
+        // console.log("Res", res);
+        const responseData = res.data[0];
+
+        console.log("responseData", responseData);
+
+        setSelectedInfo(responseData);
+        // setCity("Sao Paulo"); //           //It changes city state and gets new data from the API
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed